### DISCUSSION TITLE
getTextBounds only works with which fonts?

### DISCUSSION THREAD

**Discussion by RiaanAspeling** [moononournation/Arduino_GFX#282] [Q&A]
Good day all.

It would seem that the library currently doesn't implement getTextBounds with u8g2Font's? Are there any other fonts that would work or has someone added the code to make it work?

Any suggestions and help would be appreciated.

**Comment by moononournation** [comment]
yes, it is not supporting u8g2 yet. I have planned start implement it once someone find this issue.

**Reply by RiaanAspeling** [reply]
It would be great if this is added. Thank you!

**Comment by RiaanAspeling** [comment]
For a quick fix I wrote a small function in my code:

```
int getTextOutputWidth(String text, const uint8_t *font) {
  U8G2_NULL u8g2(U8G2_R0);
  u8g2.setFont(font);
  return u8g2.getStrWidth(text.c_str());
}
```
And just call it with:
```
int width = getTextOutputWidth("Some text", u8g2_font_inb24_mr);
```

**Reply by moononournation** [reply]
![image](https://user-images.githubusercontent.com/13228891/230428143-9faab286-4203-4696-8557-4bc479bb7474.jpeg)
Just commited the implementation, please get update and try.
